### PR TITLE
[POT] Fix bug with git

### PR DESCRIPTION
**Pull request by l-bat**

### CODE REVIEW

**Comment by AlexKoff88** [comment]
@l-bat, are you able to reproduce the issue on the current version and check that your fix works?

File: tools/pot/setup.py [hunk 1]
```diff
@@ -16,17 +16,22 @@
...
     except Exception: # pylint:disable=W0703
         return "unknown version"
 
 
 def get_version():
     version = generate_pot_version()
-    if version == "unknown version":
-        version_txt = os.path.join(os.path.dirname(os.path.realpath(__file__)), "version.txt")
+    version_txt = os.path.join(os.path.dirname(os.path.realpath(__file__)), "version.txt")
+    if version != "unknown version":
```
Comment (azhogov) [inline tools/pot/setup.py:31]: "unknown version" should be defined as constant or None or True/False can be used

File: tools/pot/setup.py [hunk 2]
```diff
@@ -16,17 +16,22 @@
 
 def generate_pot_version():
     try:
-        branch_name = subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]).strip().decode()
-        commit_hash = subprocess.check_output(["git", "rev-parse", "HEAD"]).strip().decode()
-        return "custom_{}_{}".format(branch_name, commit_hash)
+        pot_dir = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(os.path.realpath(__file__))),
+                                                "openvino", "tools"))
+        branch_name = subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"], cwd=pot_dir)
```
Comment (azhogov) [inline tools/pot/setup.py:21]: it seems duplication

Comment (l-bat) [reply tools/pot/setup.py:21]: Yes, but we can't import anything in `setup.py` from `openvino.tools.pot `

**Review by azhogov — APPROVED** [review]

**Review by AlexKoff88 — APPROVED** [review]

**Comment by AlexKoff88** [comment]
@l-bat, please rebase to pass the CI.

**Pull request merged**